libraries/AP_Math/location.cpp

```diff
@@ -23,14 +23,16 @@
 #include "AP_Math.h"
 #include "location.h"
 
-// return bearing in centi-degrees between two positions
+// return bearing_rad in radians between two positions
+float get_bearing_rad(const Vector2f &origin, const Vector2f &destination)
+{
+    return wrap_2PI(atan2f(destination.y-origin.y, destination.x-origin.x));
```

Comment:
I don't think we need this helper at all, better to just use the `angle` method on the Vector2.

Reply:
the difference between using this function and angle is just that angle will return -PI ~ +PI while this function returns 0 ~ 2xPI().  We probably handle that correctly but if we're using this function as a straight substitution for the existing get_bearing_cd() we should probably note change the output range (e.g. let's keep the function)